hydra_provider/src/database_service/hydra_input/messages/log.rs

```diff
@@ -0,0 +1,22 @@
+use sqlx::{query, Postgres, Transaction, postgres::PgQueryResult, Error};
+use messages::Log;
+use crate::database_service::hydra_input::saveable::SaveableData;
+
+impl SaveableData for Log {
+	async fn save(
+		&self,
+		transaction: &mut Transaction<'_, Postgres>,
+		rocket_message_id: i32,
+	) -> Result<PgQueryResult, Error> {
+		query!(
+			"INSERT INTO rocket_log
+				(rocket_message_id, level, event)
+				VALUES ($1, $2, $3)",
+			rocket_message_id,
+			format!("{:?}", self.level),
+			format!("{:?}", self.event)
```

Comment:
```suggestion
			format!("{}", self.event)
```

Not sure if this was tested and working with a Debug, but pretty sure Display should be used here.

Reply:
I forwarded this from the old code. I'm guessing it was tested? But we'll double check when we test this weekend :)